schema.go

```diff
@@ -951,7 +951,11 @@ func makeNodeOf(t reflect.Type, name string, tag []string) Node {
 	if node.Repeated() && !list {
 		elemKind := node.GoType().Elem().Kind()
 		if elemKind == reflect.Slice {
-			panic("unhandled nested slice on parquet schema without list tag")
+			// Special case: allow strings in a logical map
+			isUint8 := node.GoType().Elem().Elem().Kind() == reflect.Uint8
+			if !isUint8 {
```

Comment:
Very minor, we're doing `GoType().Elem()` twice in this case, we could capture `elemType` on the first invocation to avoid that.

That would also improve readability a bit, the double `.Elem()` was a bit hard to parse.

Reply:
Thanks for the suggestion; I've revised it. I'm personally less worried about the difference between these versions than about the possibility that the resulting semantics are meaningless. Does this check reflect a parquet-go limitation, or a parquet format limitation?

If it's a library limitation, fine, extended cases like `[][][]uint8` should still be blocked, and hopefully it's okay to treat `[][]uint8` as `[]string`. But if it's a format limitation, it seems like this is more of an assertion that I don't deeply understand; does repeated inherently apply only to non-repeated types with a singular exception for []uint strings?

Reply:
The assertion here is about requiring nested lists to be represented with the logical list type in parquet, because the format doesn't allow for inlined "repeated repeated" constructs.

I don't think the code as it is today would treat `[][]uint8` as `[]string`, am I missing something?

Reply:
Pardon my imprecision (and confusion). A string is represented by a physical `[]uint8`, so repeated strings are represented by `[][]uint8`. Something about altering (*columnLoader).open to use mapType instead of groupType causes TestReader/map_of_repeated_values to panic where it did not before. This test models a `map[utf8string][]utf8string` where utf8string is a defined type with underlying type of string.

This change allows that test to complete successfully. Removing the entire stanza instead does so as well, but it causes TestIssue327 to fail because it doesn't panic (see segmentio/parquet-go#327 / segmentio/parquet-go#331).

Reply:
OK, if all tests pass we don't have regressions that we know about, let's go 👍